static/js/themeSwitcher.js

```diff
@@ -12,7 +12,6 @@
 
   function setTheme(theme) {
     document.querySelector('html').setAttribute(themeAttr, theme);
-    window.localStorage.setItem('theme', theme);
```

Comment:
I would expect with this line removed that the last selected theme color would no longer be remembered by the browser, but a quick test (assuming I did it correctly) shows that it is in fact being remembered. Is localstorage being set somewhere else and this line is a duplicate or something?

Reply:
There is another `window.localStorage.setItem('theme', theme);` in setTheme in main.js.

`setTheme` and `getTheme` in `themeSwitcher.js` are called when the page is first loaded.

There are separate setTheme and getTheme functions in main.js that are called when the theme toggle is clicked.

You can verify this behaviour by putting logging statements in the set and get theme functions in both files. setTheme in themeSwitcher.js is called once on loading, `setTheme` in main.js is called when the theme toggle is clicked.

So if someone visits for the first time or clears cookies/cache, the initial theme will match their OS/browser theme, but with this change, that preference won't be stored. The event listener will also change the theme without having to refresh the page.

Once someone clicks the theme toggle button, however, the theme preference will be saved and the theme will become static.